### PR TITLE
Improve locks PROPFIND

### PR DESCRIPTION
**Pull request by CarlSchwan**

### CODE REVIEW

File: lib/Tools/Db/ExtendedQueryBuilder.php [hunk 1]
```diff
@@ -592,7 +592,7 @@ public function exprLimitInArray(string $field, array $values, string $alias = '
 
 		$expr = $this->expr();
 
-		return (string)$expr->in($field, $this->createNamedParameter($values, IQueryBuilder::PARAM_STR_ARRAY));
+		return (string)$expr->in($field, $this->createNamedParameter($values, IQueryBuilder::PARAM_INT_ARRAY));
```
Comment (CarlSchwan) [inline lib/Tools/Db/ExtendedQueryBuilder.php:595]: TODO I should probably create a new method for this instead of overwriting this method

Comment (juliushaertl) [reply lib/Tools/Db/ExtendedQueryBuilder.php:595]: Yes, definitely :)

Comment (ArtificialOwl) [reply lib/Tools/Db/ExtendedQueryBuilder.php:599]: and put back `PARAM_STR_ARRAY` !

**Review by juliushaertl — APPROVED** [review]

File: lib/Tools/Db/ExtendedQueryBuilder.php (hunk 1 above)
Comment (juliushaertl) [reply lib/Tools/Db/ExtendedQueryBuilder.php]: Why? We are matching against integers in the database for the file id

**Pull request merged**